### ISSUE TITLE
1010 EZ submission failure email

### ISSUE THREAD

**Issue by aliyahblackmore** [department-of-veterans-affairs/va.gov-team#94300]
## Issue description
The 1010 EZ product team needs content support on their silent failure email. This email is sent to Veterans if their form submission fails.
[Intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/93979)

---
## Tasks
- [x] Assess request
- [x] Draft content
- [x] Share with requesting team

## Acceptance Criteria
- [x] Feedback has been shared with requesting team

## Configure this issue
- [ ] _Attach to an Epic (what body of work is this a part of?)_
- [ ] _Label with Originator/Team (product team or stakeholder requesting support)_
- [ ] _Label with Open Date_
- [ ] _Label with Estimate (level of effort expected for this ticket)_
- [ ] _Add assignee name to each task they will complete (if known)_

**Comment by aliyahblackmore** [comment]
**Update here:** 

- Chatted with Danielle on 10/2
- I'll move forward with the content recommendations
- I'll share the updates with the product team before EOD tomorrow

**Comment by aliyahblackmore** [comment]
**Update here:** 

The content recommendations/adjustments are in their [Figma file](https://www.figma.com/design/UljiHam46o5DItC5iDgmPd/10-10EZ?node-id=6579-432686&t=tVY4YRh1jXNvInmi-4).

And I let the team know that this content recommendations are closely aligned with some of VA notify template work CAIA is working on.

**Comment by aliyahblackmore** [comment]
**Update here:**

- The team shared new information yesterday about the recovery steps - all Veterans who provided their phone number will get a call about 2 days after they get the silent failure email.

- Based on the recovery updates from the team, I adjusted the content [in Figma](https://www.figma.com/design/tggcJk382w9yQ0ElwKfh3N/10-10EZR?node-id=5340-1226&t=gRWwEBpTYizaFow4-4).

- I removed the "Submission details" box and adjusted the signature. 

Heather is going to confirm whether Veterans who signed in to submit their for will need to fill out the entire form again.This is ready for the team and any necessary HEC SME accuracy reviews.
